pkg/agent/params.go

```diff
@@ -176,11 +175,7 @@ func assignKubernetesParameters(properties *datamodel.Properties, parametersMap
 				addValue(parametersMap, "kubeBinaryURL", kubernetesConfig.CustomKubeBinaryURL)
 			}
 
-			kubernetesHyperkubeSpec := hyperkubeImageBase + k8sComponents["hyperkube"]
-			if kubernetesConfig.CustomHyperkubeImage != "" {
-				kubernetesHyperkubeSpec = kubernetesConfig.CustomHyperkubeImage
-			}
```

Comment:
reminder: don't forget to move this logic to RP side

Reply:
Yes, next time we vendor in AgentBaker, I'll go into that branch apply the same logic to generate the strings.